src/encryption.ts

```diff
@@ -177,6 +177,9 @@ export function decryptOnRead<Models extends string, Actions extends string>(
     }) {
       try {
         if (!cloakedStringRegex.test(cipherText)) {
+          if (fieldConfig.strictDecryption) {
+            throw new Error('Value is not encrypted and mode=strict')
```

Comment:
**suggestion** Could you move the error message to the `errors.ts` file, to add some context (model/field) please? Having this in logs helps a lot debugging those kind of errors.

Reply:
Wouldn't this be wrapped in the FieldDecryptionError in the catch?